tests/testsuite/kubevirtresource.go

```diff
@@ -107,6 +107,7 @@ func AdjustKubeVirtResource() {
 		virtconfig.VMExportGate,
 		virtconfig.KubevirtSeccompProfile,
 		virtconfig.HotplugNetworkIfacesGate,
+		virtconfig.VMPersistentState,
```

Comment:
Do we want to enable this fg for the whole test suite? Or do we want to restrict it to the only persistent vTPM tests?